feat: add Microsoft Copilot to fleet comparison page (#3874)
langchain-ai/docs: src/langsmith/fleet/comparison.mdx

<div className="compact-first-col">

| **Aspect** | **LangSmith Fleet** | **Claude Cowork** | **Amazon Quick** | **Google Workspace Studio** | **Microsoft Copilot** |
| - | ------------------- | ----------------- | ---------------- | --------------------------- | --------------------- |
| **Primary use case** | Teams building purpose-built agents to share across an organization, with no-code creation and code export for custom deployments; individuals using a general-purpose chat agent for any task | Individual desktop knowledge work | Enterprise AI with AWS data integration | No-code agents for Google Workspace | Low-code agents for Microsoft 365 |
| **Model support** | Model-agnostic: any LLM with an OpenAI-compatible or Anthropic-compatible API | Claude only | — | Gemini 3 | Curated OpenAI + Anthropic models; bring-your-own via Azure AI Foundry |

> | **Interface** | Web app, Slack app, Teams app, API | Desktop, mobile, Slack, M365 connectors | Web, desktop, browser extensions, Slack, Teams | Web app, Gmail and Chat sidebars | Copilot Studio, Teams, M365 Copilot, websites |

Reviewer: websites is vague

```suggestion
| **Interface** | Web app, Slack app, Teams app, API | Desktop, mobile, Slack, M365 connectors | Web, desktop, browser extensions, Slack, Teams | Web app, Gmail and Chat sidebars | Teams, M365 apps, web, mobile, Windows, Copilot Studio |
```

**Fleet** also lets you set tool-level approval requirements so agents check with you before executing sensitive steps, with a [centralized inbox](https://smith.langchain.com/agents/inbox) for reviewing, editing, and approving actions. No other platform in this comparison offers a single centralized approvals inbox spanning all agents.

<div className="compact-first-col">

| Feature | **Fleet** | **Claude Cowork** | **Amazon Quick** | **Google Workspace Studio** | **Microsoft Copilot** |
| ------- | --------- | ----------------- | ---------------- | --------------------------- | --------------------- |
| General-purpose chat agent | ✅ [Fleet chat](https://smith.langchain.com/agents) | ✅ | ✅ | ❌ | ✅ |
| No-code agent builder | ✅ | ❌ | ✅ | ✅ | ✅ |

> | Slack-native integration | ✅ [Native Slack app](/langsmith/fleet/slack-app) | ✅ | ✅ | ⚠️ | ⚠️ |

Reviewer: ```suggestion
| Slack-native integration | ✅ [Native Slack app](/langsmith/fleet/slack-app) | ✅ | ✅ | ⚠️ | ⚠️  (via Azure Bot Service) |
```

<div className="compact-first-col">

| Feature | **Fleet** | **Claude Cowork** | **Amazon Quick** | **Google Workspace Studio** | **Microsoft Copilot** |
| ------- | --------- | ----------------- | ---------------- | --------------------------- | --------------------- |
| Role-based access control | ✅ [RBAC with per-tool permissions](/langsmith/rbac) | ✅ | ✅ | ✅ | ✅ |
| Attribute-based access control | ✅ [Per MCP server and integration](/langsmith/fleet/access-and-oversight#attribute-based-access-control) | ❌ | ❌ | ❌ | — |
| Per-agent sharing and permissions | ✅ [Clone, Run, and Edit access per agent](/langsmith/fleet/access-and-oversight#permissions-and-sharing) | ⚠️ | ✅ | ⚠️ | ✅ |
| Credential model (fixed or per-user) | ✅ [Configurable per agent](/langsmith/fleet/access-and-oversight#agent-identity-and-credentials) | ✅ | ✅ | ✅ | ✅ |
| Spend limits | ⚠️ Managed at workspace level | ✅ | ⚠️ | ⚠️ | ✅ |
| SCIM provisioning | ✅ | ✅ | ✅ | — | ✅ |
| Audit trail | ✅ [Structured LangSmith traces](/langsmith/fleet/access-and-oversight#observability-and-audit-trail) | ✅ | ✅ | ✅ | ✅ |

</div>

### Model flexibility

> **Fleet** supports any LLM via the OpenAI or Anthropic chat spec, including models from OpenAI, Anthropic, Google, and self-hosted providers. Most other platforms are locked to a single vendor's models or offer limited model selection. **Fleet** is also ecosystem-agnostic, while other platforms require adoption of a specific cloud or productivity suite.

Reviewer: ```suggestion
**Fleet** supports any LLM via the OpenAI or Anthropic chat spec, including self-hosted providers, with no ecosystem dependency. Microsoft Copilot offers curated multi-vendor models and a bring-your-own path via Azure AI Foundry, but full flexibility requires Azure infrastructure. Google Workspace Studio and Amazon Quick are more constrained to their respective vendor ecosystems.

Of the platforms compared here, only Fleet works with any OpenAI- or Anthropic-compatible API endpoint regardless of cloud provider.
```

| Thread-scoped context | ✅ | ✅ | ✅ | ✅ | ✅ |
| Self-updating agents | ✅ [Agents can add tools, remove tools, and update their own instructions](/langsmith/fleet/essentials#self-updates) | ❌ | ❌ | ❌ | ❌ |
| Approval gate for memory writes | ✅ [Configurable per agent](/langsmith/fleet/manage-agent-settings) | ❌ | ❌ | ❌ | — |

</div>

### Observability and governance

**Fleet's** clearest advantage is its native connection to LangSmith. Every agent run is traced in LangSmith, making it easy to debug performance and run evaluations at scale. Other platforms offer basic logging and audit trails, but none match Fleet's depth of LLM-aware tracing, evaluations, and debugging through a dedicated observability platform.

<div className="compact-first-col">

| Feature | **Fleet** | **Claude Cowork** | **Amazon Quick** | **Google Workspace Studio** | **Microsoft Copilot** |
| ------- | --------- | ----------------- | ---------------- | --------------------------- | --------------------- |
| Native tracing | ✅ [LangSmith traces for every run](/langsmith/observability) | ✅ | ⚠️ | ⚠️ | ⚠️ |

> | Evaluations | ✅ [LangSmith evaluations](/langsmith/evaluation-concepts) | ❌ | ❌ | ❌ | ✅ |

Reviewer: The ✅ for Microsoft Copilot in the Evaluations row is accurate in the sense that a real, documented evaluation system exists. But the two are not equivalent in depth or flexibility — LangSmith is a dedicated observability and evaluation platform, while Copilot Studio's evals are a testing feature within an agent-builder UI. If the table implied parity, a ⚠️ might be more precise, depending on how the table is meant to communicate relative capability rather than just presence/absence.

```suggestion
| Evaluations | ✅ [LangSmith evaluations](/langsmith/evaluation-concepts) | ❌ | ❌ | ❌ | ⚠️ |
```

Reviewer 2: will change to ! since we want to make ourselves look differentiated a bit

<div className="compact-first-col">

| Feature | **Fleet** | **Claude Cowork** | **Amazon Quick** | **Google Workspace Studio** | **Microsoft Copilot** |
| ------- | --------- | ----------------- | ---------------- | --------------------------- | --------------------- |
| Google Workspace (Gmail, Drive, Sheets, Docs) | ✅ | ✅ | ⚠️ | ✅ | ⚠️ |
| Microsoft 365 (Outlook, Teams, SharePoint, Excel) | ✅ | ✅ | ✅ | ❌ | ✅ |
| GitHub | ✅ | ✅ | ✅ | — | — |
| Slack | ✅ [Native](/langsmith/fleet/slack-app) | ✅ | ✅ | ⚠️ | ❌ |
| CRM (Salesforce, HubSpot) | ✅ | — | ✅ | ⚠️ | ✅ |
| Project management (Linear, Jira, Notion) | ✅ | ✅ | ✅ | ⚠️ | ⚠️ |
| Custom tools via MCP | ✅ | ✅ | ✅ | ❌ | ✅ |
| Webhooks | ✅ [Webhooks](/langsmith/fleet/webhooks) | ❌ | ❌ | ⚠️ | ✅ |

</div>

For pricing and SLA information, [contact sales](https://www.langchain.com/contact-sales).

<Note>

>     Last updated May 5th, 2026. These products evolve quickly. If something has changed, please [file an issue](https://github.com/langchain-ai/docs/issues) to help us keep this page current.

Reviewer: per Google style
```suggestion
    Last updated May 5, 2026. These products evolve quickly. If something has changed, please [file an issue](https://github.com/langchain-ai/docs/issues) to help us keep this page current.
```
